Raven.Database/Config/MemoryStatistics.cs

```diff
@@ -111,8 +111,7 @@ static MemoryStatistics()
                             return;
                         case 2: // LowMemorySimulationEvent
                             Log.Warn("Low memory simulation, will try to reduce memory usage...");
-
-                            RunLowMemoryHandlers("Simulated low memory");
+                            RunLowMemoryHandlers("System detected low memory");
```

Comment:
While I merged this, I think this need more eyeballs.
The issue is that we are reporting this for the client as part of insight to how the heuristics work, so we want this to be accurate.